samples/net/ipv4_autoconf/prj_qemu_x86.conf

```diff
@@ -0,0 +1,37 @@
+CONFIG_NETWORKING=y
```

Comment:
It looks like you should be able to merge the prj files together and just have one prj.conf file. Note that you do not need to mention CONFIG_NET_SLIP_TAP in conf file as that will be automatically applied if compiled for qemu.